schema/schema_test.go

```diff
@@ -0,0 +1,1291 @@
+/*
+ * Copyright 2016-2022 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package schema
+
+import (
+	"context"
+	"reflect"
+	"sync"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/protos/pb"
+	"github.com/dgraph-io/dgraph/tok"
+	"golang.org/x/net/trace"
+)
+
+type fields struct {
+	RWMutex   sync.RWMutex
+	predicate map[string]*pb.SchemaUpdate
+	types     map[string]*pb.TypeUpdate
+	elog      trace.EventLog
+	mutSchema map[string]*pb.SchemaUpdate
+}
+
+func TestGetWriteContext(t *testing.T) {
+	type args struct {
+		ctx context.Context
+	}
+	tests := []struct {
+		name string
+		args args
+		want context.Context
+	}{
+		{
+			name: "TestGetWriteContext",
+			args: args{
+				ctx: context.Context(context.Background()),
+			},
+			want: context.WithValue(context.Background(), isWrite, true),
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := GetWriteContext(tt.args.ctx); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("GetWriteContext() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_state_DeleteAll(t *testing.T) {
```

Comment:
Can you standardize the test function names to camel case (TestFooBar) or the also acceptable mixed (Test_FooBar)?

Reply:
+1 this is a Go convention, in our repo we tend to use pure camel case (e.g. TestExampleOne)